verification/api/handler.go

```diff
@@ -320,9 +335,35 @@ func (o *Handler) SubmitEvidence(c *gin.Context) {
 		return
 	}
 
+	// read body (i.e., evidence)
+	evidence, err := io.ReadAll(c.Request.Body)
+	if err != nil || len(evidence) == 0 {
+		o.logger.Error("unable to read evidence from the request body: %v", err)
+		ReportProblem(c,
+			http.StatusBadRequest,
+			"unable to read evidence from the request body",
+		)
+		return
+	}
+
 	// read content-type and check against supported attestation formats
 	mediaType := c.Request.Header.Get("Content-Type")
 
+	if isCMW(mediaType) {
+		var w cmw.CMW
+
+		if err := w.Deserialize(evidence); err != nil {
```

Comment:
cmw Deserialize method needs a pointer to CMW object to set the fields.

Reply:
I meant : func (o *CMW) Deserialize(b []byte) error

Reply:
as discussed in person, the code is ok as-is.